prow/scripts/cluster-integration/kyma-cli-alpha-upgrade.sh

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env bash
+
+#Description: TEMPORARY PIPELINE FOR ALPHA FEATURES TESTING. WORK IN PROGRESS. Related issue: https://github.com/kyma-project/test-infra/issues/3057
+#
+#
+#Expected vars:
+#
+# - KYMA_PROJECT_DIR - directory path with Kyma sources to use for installation
+# - GARDENER_REGION - Gardener compute region
+# - GARDENER_ZONES - Gardener compute zones inside the region
+# - GARDENER_KYMA_PROW_KUBECONFIG - Kubeconfig of the Gardener service account
+# - GARDENER_KYMA_PROW_PROJECT_NAME Name of the gardener project where the cluster will be integrated.
+# - GARDENER_KYMA_PROW_PROVIDER_SECRET_NAME Name of the GCP secret configured in the gardener project to access the cloud provider
+# - MACHINE_TYPE (optional): GCP machine type
+#
+#Permissions: In order to run this script you need to use a service account with permissions equivalent to the following GCP roles:
+# - Compute Admin
+# - Service Account User
+# - Service Account Admin
+# - Service Account Token Creator
+# - Make sure the service account is enabled for the Google Identity and Access Management API.
+
+set -e
+
+readonly GARDENER_CLUSTER_VERSION="1.16"
+
+#Exported variables
+export TEST_INFRA_SOURCES_DIR="${KYMA_PROJECT_DIR}/test-infra"
+export TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS="${TEST_INFRA_SOURCES_DIR}/prow/scripts/cluster-integration/helpers"
+
+# shellcheck source=prow/scripts/lib/gardener/gcp.sh
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/gardener/gcp.sh"
+# shellcheck disable=SC1090
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/testing-helpers.sh"
+# shellcheck source=prow/scripts/lib/utils.sh
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/utils.sh"
+
```

Comment:
Direct source of log library would be better than relying on other libraries to import it
```bash
# shellcheck source=prow/scripts/lib/log.sh
source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/log.sh"
```